### ISSUE TITLE
Bug / Feature Request - Automatic Action: Asteroid Destiny Draws

### ISSUE THREAD

**Issue by fungineer-86** [PlayersCommittee/gemp-swccg-public#245]
When Asteroid Destinies as Drawn (especially as an automatic action at the end of the Control phase) it becomes unclear which ship is being targeted for each asteroid destiny draw. 
![image](https://user-images.githubusercontent.com/72903915/100576983-d84ecb80-32a4-11eb-8fc8-ecfa3706a466.png) This becomes more important when different ships are at different locations, or if two (non-unique) ships have weapons/pilots/passangers/etc. upon them. When the image comes up, there's no way to know which ship is being targeted vs. which one is not, until after the draw resolves. 

Expected situation - the automatic actions display on the table (or the pop-up displays all cards related to the shown ship + it's current location/related system). 

https://forum.starwarsccg.org/viewtopic.php?f=1105&t=76318&p=1312915&hilit=asteroid#p1312915

**Comment by BOTVHD** [comment]
Yeah, if it is left to an automatic action, the simultaneous action stuff kicks in and you have to choose which order to carry these out.  I wonder if it can be re-worked without the pop-up and just force you to pick a valid ship on table (highlighted) repeatedly until all of the asteroid draws are complete.

**Comment by BOTVHD** [comment]
Same issue happens (I believe) with choosing between multiple Timer Mine explode actions at the start of a turn.  There isn't a great way to know which one is being picked (and it can be important to pick accurately, if a destiny was tracked!).
